apps/accounts/mfa_forms.py

```diff
@@ -10,7 +10,11 @@ class MFACodeForm(forms.Form):
 
 
 class LoginForm(forms.Form):
-    username = forms.CharField(max_length=30, label=_('Username'))
-    password = forms.CharField(widget=forms.PasswordInput, max_length=120,
+    username = forms.CharField(max_length=150, label=_('Email'))
+    password = forms.CharField(widget=forms.PasswordInput, max_length=150,
                                label=_('Password'))
     required_css_class = 'required'
+
+    def clean_username(self):
```

Comment:
Is this function actually being called anywhere? I'm not sure this is actually used.

Reply:
@samgensburg-gov yes this is being used....called when form is validated....forces it to lower

Reply:
yes. called on form validation.